### ISSUE TITLE
Mapping values include comments

### ISSUE THREAD

**Issue by j3lamp** [jbeder/yaml-cpp#306]
If a block style mapping value is followed by a line containing only a comment which is preceded by at least one space the comment will be included as part of the value.

For example the YAML
```yaml
---
key: value
 # comment
...
```
will have the value `value # comment` for `key`.

It would be expected that the value would be `value`. If there is no space in front of the comment then the YAML document is parsed as expected.

This was tested on Mac OS X 10.8 and 10.9 and Fedora 19.

Note:
These parse as expected:
```
---
key: value
# comment
...
```
and
```
---
key: value # comment
...
```

Most surprisingly this parses incorrectly:
```
---
key: value

 # comment
...
```
Yielding a value of `value\n# comment`.

**Comment by jbeder** [comment]
Thanks for the report.

**Comment by jbeder** [comment]
Fixed 7d2873ce9f2202ea21b6a8c5ecbc9fe38032c229

**Comment by j3lamp** [comment]
Awesome, thanks!
